### ISSUE TITLE
Need to sign the addon to work on Firefox

### ISSUE THREAD

**Issue by Amustache** [hestiaAI/DescriCookie#1]
cf. https://stackoverflow.com/questions/68472808/cannot-install-own-developed-firefox-extension-through-manage-your-extension-b

**Comment by mvidonne** [comment]
@Amustache 
please check the previous version of the AdRadar extension
https://github.com/hestiaAI/ad-radar/commit/1074886720a511df42018bba14251c77c2df4c06

we could use the xpi file through
Settings > Extensions & Themes > Tools for all add-ons > Debug Add-ons > Load Temporary Add-on

**Comment by Amustache** [comment]
@mvidonne Thank you! It is indeed what I added in the README!

**Comment by mvidonne** [comment]
@Amustache 
Yes, but the link to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Your_first_WebExtension#installing is the README was not clear. I would prefer a step by step description of what I have to do.
